ls_extstorage.ino

```diff
@@ -2132,3 +2280,73 @@ void copyConfigurationV15(void* target, void* source) {
 
   memcpy(&t->project, &s->project, sizeof(SequencerProject));
 }
+
+/*************************************************************************************************/
+
+void copyConfigurationVLatest(void* target, void* source) {
+  Configuration* t = (Configuration*)target;
+  ConfigurationVLatest* s = (ConfigurationVLatest*) source;
+
+  memcpy(&t->device, &s->device, sizeof(s->device));
+  t->device.version = 16 + MICROLINN_VERSION_OFFSET;
+
+  memcpy(&t->settings.global, &s->settings.global, sizeof(s->settings.global));
+  for (int split = 0; split < NUMSPLITS; split++) {
+    memcpy(&t->settings.split[split], &s->settings.split[split], sizeof(s->settings.split[split]));
+  }
+
+  for (byte p = 0; p < NUMPRESETS; p++) {
+    memcpy(&t->preset[p].global, &s->preset[p].global, sizeof(s->preset[p].global));
+    for (int split = 0; split < NUMSPLITS; split++) {
+      memcpy(&t->preset[p].split[split], &s->preset[p].split[split], sizeof(s->preset[p].split[split]));
+    }
+  }
+  memcpy(&t->project, &s->project, sizeof(s->project));
+}
+
+void copyConfigurationMicroLinnV72A(void* target, void* source) {
```

Comment:
This is the same as the function above, other than the type of the "source" config, and the placeholder at the end for microlinn-specific data conversions.